infra/resources/_modules/com_storage_account/storage_account_io_com.tf

```diff
@@ -0,0 +1,22 @@
+module "dx-azure-storage-account" {
+  source  = "pagopa/dx-azure-storage-account/azurerm"
+  version = "0.0.9"
+
+  subnet_pep_id       = var.subnet_pep_id
+  tags                = var.tags
+  tier                = "s"
+  environment         = merge(var.environment, { app_name = "messages" })
```

Comment:
I would remove the `app_name` to get `iopitncomst` as resource name (if `app_name` can't be empty, remove the `domain` and set `com` as `app_name`)

Reply:
[8ecfb59](https://github.com/pagopa/io-messages/pull/107/commits/8ecfb593036761f7de00d340f4bfbc37c29ee9bb)